### PR TITLE
Feature/create detail introduction

### PR DESCRIPTION
**Pull request by CBinh710**
https://github.com/users/CBinh710/projects/2/views/1?pane=issue&itemId=72090219
image

### CODE REVIEW

File: practice-one/src/assets/images/play-button.png
Comment (hungdo-agilityio) [inline practice-one/src/assets/images/play-button.png:1]: Recheck the extension of this image and other icon

File: practice-one/src/components/introduction.html [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+  <div class="introduction-content-container">
+
+     <div class="people-trust-icon">
+        <img src="./assets/images/trust.png" alt="Heart icon" />
+        <p>People Trust us</p>
+     </div>
+     <div class="header-introduction"><p>We're <span style="color:var(--color-primary);">Serious</span> For <br><span style="color:var(--color-primary);">Food</span> & <span style="color: var(--color-accent);">Delivery</span>.</p></div>
+     <div class="header-decor"><img src="./assets/images/decor.png" alt="decor flash"></div>
+     <div class="para-introduction"><p>Best cooks and best delivery guys all at your service. Hot tasty food will reach you in 60 minutes.</p></div>
+ </div>
```
Comment (hungdo-agilityio) [inline practice-one/src/components/introduction.html]: recheck the indent for all

File: practice-one/src/components/introduction.html [hunk 2]
```diff
@@ -0,0 +1,18 @@
+<section class="introduction-container">
+
+  <div class="introduction-content-container">
+
+     <div class="people-trust-icon">
+        <img src="./assets/images/trust.png" alt="Heart icon" />
+        <p>People Trust us</p>
+     </div>
+     <div class="header-introduction"><p>We're <span style="color:var(--color-primary);">Serious</span> For <br><span style="color:var(--color-primary);">Food</span> & <span style="color: var(--color-accent);">Delivery</span>.</p></div>
```
Comment (hungdo-agilityio) [inline practice-one/src/components/introduction.html]: This hard to read, can use break each element separately line when it come many elements like this. Ex:
```
  <div>
     <p>
     <span>
      <br />
   </div>
```

Comment (hungdo-agilityio) [reply practice-one/src/components/introduction.html]: And also do not use inline style

File: practice-one/src/components/introduction.html [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+     <div class="header-decor"><img src="./assets/images/decor.png" alt="decor flash"></div>
+     <div class="para-introduction"><p>Best cooks and best delivery guys all at your service. Hot tasty food will reach you in 60 minutes.</p></div>
+ </div>
+
+ <div class="illustrate-image">
+     <img src="./assets/images/intro.png" alt="Illustra" />
```
Comment (hungdo-agilityio) [inline practice-one/src/components/introduction.html]: `Illustra` check typo

File: practice-one/src/styles/components/introduction.css [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+}
+
+.introduction-content-container {
+  padding-top: 50px;
+}
+
+.people-trust-icon {
+  display: flex;
+  text-align: center;
+  box-shadow: drop;
```
Comment (hungdo-agilityio) [inline practice-one/src/styles/components/introduction.css]: Recheck this I dont think this is properly value

File: practice-one/src/components/introduction.html (hunk 2 above)
Comment (hungdo-agilityio) [reply practice-one/src/components/introduction.html]: Why not use heading tag

File: practice-one/src/styles/components/introduction.css (hunk 4 above)
Comment (CBinh710) [reply practice-one/src/styles/components/introduction.css]: i fixed it

File: practice-one/src/components/introduction.html (hunk 3 above)
Comment (CBinh710) [reply practice-one/src/components/introduction.html]: thanks

File: practice-one/src/components/introduction.html (hunk 2 above)
Comment (CBinh710) [reply practice-one/src/components/introduction.html]: I changed it , thanks for your contribution

File: practice-one/src/assets/images/play-button.png
Comment (CBinh710) [reply practice-one/src/assets/images/play-button.png:1]: i understood and changed it, thanks for all

File: practice-one/src/components/introduction.html [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+     </div>
+     <h1 class="header-introduction">
+        <p>
+        We're 
+        <span>Serious</span>
+        For 
+        <br/>
+        <span>Food</span> 
+        & 
+        <span>Delivery</span>.
```
Comment (hungdo-agilityio) [inline practice-one/src/components/introduction.html]: I saw in design has many places have color red and yellow, don't repeat yourself just create a class for it

File: practice-one/src/styles/variables.css [hunk 6]
```diff
@@ -1,11 +1,14 @@
...
+  --color-icon-wrapper:#FEE5E5;
   --color-accent: #FDC55E;
   --color-light: #FFFFFF;
   --color-neutral: #C4C4C4;
   --color-warning: #FFB800;
   --color-gradient-warm: linear-gradient(0deg, rgba(245, 71, 72, 0) 57.72%, #FDC55E 112.46%);
   --color-gradient-strong: linear-gradient(179.69deg, rgba(255, 255, 255, 0) , #F54748 303.75%);
   --font-family-primary: 'Sofia Pro Regular';
+  --font-family-dark: 'Sofia Pro Black';
+  --font-family-search-box: 'Poppins Medium';
```
Comment (hungdo-agilityio) [inline practice-one/src/styles/variables.css]: This is wrong, you no need to define another fontfamily everytime fontweight change

Comment (hungdo-agilityio) [reply practice-one/src/styles/variables.css]: all you need is
```
--font-family-primary: 'Sofia'
--font-family-secondary: 'Poppins'

.example-class {
  font-family: var(--font-family-primary);
  font-weight: medium
}
```

Comment (CBinh710) [reply practice-one/src/styles/variables.css]: i updated it

File: practice-one/src/components/introduction.html (hunk 5 above)
Comment (CBinh710) [reply practice-one/src/components/introduction.html]: i did it

File: practice-one/src/styles/variables.css [hunk 7]
```diff
@@ -1,11 +1,13 @@
 :root {
   --color-dark: #191919;
   --color-primary: #F54748;
+  --color-icon-wrapper:#FEE5E5;
```
Comment (hungdo-agilityio) [inline practice-one/src/styles/variables.css]: recheck the format here

File: practice-one/src/components/introduction.html [hunk 8]
```diff
@@ -0,0 +1,24 @@
...
+        <b>  
+           We're 
+          <span class="primary-color"> Serious </span>
+           For 
+          <br/>
+          <span class="primary-color">Food</span> 
+           & 
+          <span class="accent-color">Delivery</span>.
+        </b>
+     </h1>
```
Comment (hungdo-agilityio) [inline practice-one/src/components/introduction.html]: This is wrong, you shouldn't wrap everything inside b tag like this

File: practice-one/src/styles/variables.css (hunk 7 above)
Comment (CBinh710) [reply practice-one/src/styles/variables.css]: i update it

File: practice-one/src/components/introduction.html (hunk 8 above)
Comment (CBinh710) [reply practice-one/src/components/introduction.html]: I understood , thanks for your comment

**Review by hungdo-agilityio — APPROVED** [review]

**Pull request merged**